recipes/pymumps/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "pymumps" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+  
+source:
+  fn: {{ version }}.tar.gz
+  url: https://github.com/santiagoropb/pymumps/archive/v{{ version }}.tar.gz
+  sha256: dddfc13eb6af56a9bcd8e42fd42c3293dc50fd155fe3847c3dd84a403edba8ba
+
+build:
+  number: 1000
+  script: python setup.py install --include-dirs $BUILD_PREFIX/include --library-dirs $BUILD_PREFIX/lib
+  skip: true  # [win]
+  
+requirements:
+  build:
+    - cython
+    - python
```

Comment:
This looks like a host section not a build section

Reply:
Sorry, as a beginner on building recipes I am still a little bit confused on what is the difference between build and host. I tried switching the keyword build to host but now I am getting an error on travis (missing CONDA_BUILD_SYSROOT). I am not sure if this is an unrelated issue. Can you suggest a work around?

Reply:
It seems that the problem is solved once setuptools is included as a run requirement.